File: Website/Sedaya/API/seni.php
<?php
require_once("../admin/syntax.php");
	header('Content-Type: aplication/json');
    $data = $syntax->view("seni,jenis where seni.jns_id=jenis.jns_id ORDER BY sn_id DESC");
if ($data) {
    $result = array();
    while ($row = mysqli_fetch_array($data)) {
        array_push($result, array(
            'sn_id'=>$row['sn_id'],
            'snm_id'=>$row['snm_id'],
            'judul'=>$row['judul'],
            'kategori'=>$row['kategori'],
            'jns_id'=>$row['jns_id'],
            'jenis'=>$row['jenis'],
            'keterangan'=>$row['keterangan'],
            'harga'=>$row['harga'],
            'gambar'=>$row['gambar'],
            'status'=>$row['status'],
        ));
    }
    echo json_encode(array(
        'message'=>'berhasil',
        'data   '=>$result
    ));
}else{
    http_response_code(404);
    echo json_encode(array(
        'code'=>'400',
        'message'=>'berhasil'
    ));
}
       
?>

File: Mobile/sedaya/app/src/main/java/com/example/sedaya/core/data/source/model/Seni.kt
package com.example.sedaya.core.data.source.model

data class Seni (
    val sn_id: String? = null,
    val snm_id: String? = null,
    val nama_snm: String?,
    val alamat: String?,
    val judul: String?,
    val kategori: String?,
    val jenis: String?,
    val keterangan: String?,
    val harga: Int?,
    val image: Int,
    val status: String?,
)